LabExT/Movement/Stages/Stage3DSmarAct.py

```diff
@@ -53,6 +56,42 @@ class Stage3DSmarAct(Stage):
         Dict of channel objects
     """
 
+    driver_loaded = MCS_LOADED
+
+    @classmethod
+    def load_driver(cls, parent) -> bool:
+        """
+        Loads driver for SmarAct by open a dialog to specifiy the driver path. This method will be invoked by the MovementWizardController.
+        """
+        driver_path_dialog = DriverPathDialog(
```

Comment:
Maybe check here if driver is already loaded before opening GUI element.

Reply:
Good idea 👍🏼  I changed it.